### PR TITLE
Modifica stato segnalazione

### PR DESCRIPTION
**Pull request by domyz21**
added "edit report status" and added a snackbar to show success or failure when you try to set/change the report status or the report priority

### CODE REVIEW

**Review by FaiellaFrancesco — APPROVED** [review]
good job, lgtm

**Review by Luigi11162 — CHANGES_REQUESTED** [review]
Resolve Conflicts, please

File: src/lib/analisi_dati/analisi_dati_gui.dart
Comment (domyz21) [inline src/lib/analisi_dati/analisi_dati_gui.dart:1]: improve formatting

File: src/lib/gestione_segnalazione_comune/gestione_segnalazione_comune_dao.dart
Comment (domyz21) [inline src/lib/gestione_segnalazione_comune/gestione_segnalazione_comune_dao.dart:1]: edit report status function

File: src/lib/gestione_segnalazione_comune/dettagli_segnalazione_comune_gui.dart
Comment (domyz21) [inline src/lib/gestione_segnalazione_comune/dettagli_segnalazione_comune_gui.dart:1]: edit status/priority report gui

**Review by domyz21 — COMMENTED** [review]
fixed merge problems

**Review by Luigi11162 — APPROVED** [review]
LGTM, conflicts resolved. Approved

**Review by benedettoscala — APPROVED** [review]

**Review by Borax-Gargamella — APPROVED** [review]

**Pull request merged**